lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -66,6 +70,16 @@ base class PolygonLayer<R extends Object>
   /// Defaults to `false`.
   final bool drawLabelsLast;
 
+  /// Color to apply to the map where not covered by a polygon
+  ///
+  /// > [!WARNING]
+  /// > On the web, inverted filling may not work as expected in some cases.
+  /// > It will not match the behaviour seen on native platforms. Avoid allowing
+  /// > polygons to intersect, and avoid using holes within polygons.
+  /// > This is due to multiple limitations/bugs within Flutter. See online
+  /// > documentation for more info.
```

Comment:
If possible, link the online resources like https://github.com/flutter/flutter/issues/124675

Reply:
This is linked from our online docs: [docs.fleaflet.dev/layers/polygon-layer#inverted-filling](https://docs.fleaflet.dev/layers/polygon-layer#inverted-filling)

Reply:
Done. Also added elsewhere.